code-postprocessing/latex-templates/templatepreamble.tex

```diff
@@ -65,7 +65,6 @@
   \begin{rotate}{90}\hspace{#1em}#2
   \end{rotate}}
 \newcommand{\DIM}{\ensuremath{\mathrm{DIM}}}
-\newcommand{\ERT}{\ensuremath{\mathrm{ERT}}}
```

Comment:
Why is \ERT is not needed anymore?

Reply:
This was just a mistake I committed without notice, fixed